archive/p/python/README.md

```diff
@@ -7,6 +7,7 @@ Welcome to Sample Programs in Python!
 - [Baklava in Python][22]
   - Solution borrowed from @erayak via the [baklava repo][21]
 - [Bubble Sort in Python][31]
+- [Capitalize in Python](capitalize.py)
```

Comment:
```suggestion
- Capitalize in Python
```

You can leave this unlinked. Otherwise, you can create a new issue for the missing article, and link it here.

Reply:
Gotcha, I misunderstood the purpose of those links. I've committed your suggested change.

Can I ask a couple questions? 
- Is re-requesting a review the right etiquette after making the requested change, or does it notify you automatically? 
- Why do (at least some of) the links point to the issues for writing the articles, rather than the completed articles? Is that intentional?